pages/collections/_type/index.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <b-container>
+    <ContentHeader
+      :title="$tc('entity.index.' + this.$route.params.type, 2)"
+    />
+    <b-row class="flex-md-row pb-5">
+      <b-col cols="12">
+        <b-card-group
+          class="card-deck-4-cols"
+          deck
+          :data-qa="`${ this.$route.params.type} listing page`"
+        >
+          <ContentCard
+            v-for="entity in entities"
+            :key="entity.id"
+            :title="entity.prefLabel[$i18n.locale]"
+            :url="entityRoute(entity)"
+            :image-url="typeof entity.isShownBy !== 'undefined' ? entity.isShownBy.thumbnail : '/'"
+            variant="mini"
+          />
+        </b-card-group>
+      </b-col>
+    </b-row>
+    <b-row>
+      <b-col>
+        <PaginationNav
+          v-if="showPagination"
+          v-model="page"
+          :limit="perPage"
+          :total-results="total"
+          :per-page="perPage"
+          :link-gen="paginationLink"
+        />
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+<script>
+  import ContentHeader from '../../../components/generic/ContentHeader';
+  import ContentCard from '../../../components/generic/ContentCard';
+  import { pageFromQuery } from '../../../plugins/utils';
+  import {
+    getEntityIndex,
+    getEntitySlug,
+    getEntityTypeHumanReadable
+  } from '../../../plugins/europeana/entity';
+
+  const PER_PAGE = 24;
+  export default {
+    name: 'CollectionTypeIndexPage',
+    components: {
+      ContentHeader,
+      ContentCard,
+      PaginationNav: () => import('../../../components/generic/PaginationNav')
+    },
+    asyncData({ query, params, redirect, error, app }) {
+      const currentPage = pageFromQuery(query.page);
+      if (!['persons', 'topics'].includes(params.type)) {
+        return  error({ statusCode: 404, message: 'unknown collection type' });
+      }
+      if (currentPage === null) {
+        // Redirect non-positive integer values for `page` to `page=1`
+        query.page = '1';
+        return redirect(app.$path({ name: 'collections-type', params: { type: params.type }, query }));
+      }
+      const entityIndexParams = {
```

Comment:
Include an additional parameter to limit the fields returned to those we need:
```js
fl: 'skos_prefLabel.*,isShownBy'
```

Reply:
adding this param all thumbnails aren't rendered/returned and I get these console errors
```
[vuex] duplicate namespace http/ for the namespaced module http
[vuex] duplicate getter key: http/canonicalUrl
[vuex] duplicate getter key: http/canonicalUrlWithoutLocale
[vuex] duplicate getter key: http/origin
```

Reply:
You are quite right.

Try this instead:
```js
fl: 'skos_prefLabel.*,isShownBy,isShownBy.thumbnail'
```

The vuex errors are unrelated. Try restarting Nuxt.